apps/system/js/airplane_mode.js

```diff
@@ -83,6 +83,10 @@
 
     _start: function() {
       this._watchList = {};
+      LazyLoader.load(['js/airplane_mode_icon.js']).then(function() {
```

Comment:
I generally agree with this, but I'm not so sure it's needed in LazyLoader cases. I do think that we should be able to trust LazyLoader to basically never error - since it's just loading files from disk, and these locations should be known. I'm sure there are people that disagree with me here though.